HW_1/task_3.py

```diff
@@ -0,0 +1,37 @@
+"""
+Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример:
+
+34
+-30
+4
+
+2
+4
+1
+
+-34
+0
+Точка на отрицательной части оси абсцисс
+"""
+def detect_coordinate_quarter(x,y):
+    if((x>0)and(y>0)):
```

Comment:
Во-первых, скобки тут не нужны. Во-вторых, если в условии с `and` присутствует повторяющиеся значения, то условие можно свернуть относительно этого значения. И в-третьих, при истинности этого условия, остальные все равно будут проверены.
```suggestion
    if x > 0 < y:
```